packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -26,17 +26,17 @@ contract AssetHolder is IAssetHolder {
      * @dev Transfers as many funds escrowed against `channelId` as can be afforded for a specific destination. Assumes no repeated entries.
      * @param fromChannelId Unique identifier for state channel to transfer funds *from*.
      * @param allocationBytes The abi.encode of AssetOutcome.Allocation
-     * @param destination External destination or channel to transfer funds *to*.
+     * @param indices Array with each entry denoting the index of a destination to transfer funds to.
      */
     function transfer(
         bytes32 fromChannelId,
         bytes calldata allocationBytes,
-        bytes32 destination
-    ) external {
+        uint256[] memory indices
```

Comment:
You should use `calldata` if you can because it is a special storage location with lower gas costs associated with it. But in practice I have often found it necessary to use `memory` for function parameters so that those functions can be used internally and externally.